modules/aws-emr-iam/main.tf

```diff
@@ -718,11 +718,11 @@ resource "aws_iam_role" "emr_ec2_instance_profile" {
   tags                 = var.tags
 }
 
-// The IAM role policy attachment(s) that attach s3 policy ARNs to the EMR EC2 iam role
-resource "aws_iam_role_policy_attachment" "emr_ec2_s3_policies" {
-  count      = length(var.s3_policy_arns)
+//IAM role policy attachment(s) that attach additional policy ARNs to the EMR EC2 iam role
+resource "aws_iam_role_policy_attachment" "emr_ec2_additional_policies" {
+  count      = length(var.additional_policy_arns)
```

Comment:
another case where `for_each` and `each.value` would be better fit.

Reply:
We replicated what we got in the S3 and Tamr VM modules.